pgx-examples/aggregate/src/lib.rs

```diff
@@ -161,15 +161,17 @@ mod tests {
         Spi::run("CREATE TABLE demo_table (value INTEGER);");
         Spi::run("INSERT INTO demo_table (value) VALUES (1), (2), (3);");
         let retval = Spi::get_one::<i32>("SELECT DEMOAVG(value) FROM demo_table;")
-            .expect("SQL select failed");
+            .expect("SQL select failed")
+            .unwrap();
         assert_eq!(retval, 2);
     }
     #[pg_test]
     fn test_integer_avg_with_null() {
         Spi::run("CREATE TABLE demo_table (value INTEGER);");
         Spi::run("INSERT INTO demo_table (value) VALUES (1), (NULL), (3);");
         let retval = Spi::get_one::<i32>("SELECT DEMOAVG(value) FROM demo_table;")
-            .expect("SQL select failed");
+            .expect("SQL select failed")
+            .unwrap();
```

Comment:
Adding double-unwraps is really grody, is there any chance we could instead have one of the `Error`'s variants indicate the `Null` case?

Reply:
I guess one can see Null as a type mismatch if the requested type is not `Option<T>`.

Reply:
```rust
let i = Spi::get_one::<i32>("SELECT NULL::integer");
```

That ought to return an Err, yes.

Reply:
Fixed.